ChatQnA/serving/tgi_gaudi/launch_tgi_service.sh

```diff
@@ -16,8 +16,8 @@
 
 # Set default values
 default_port=8080
-default_model="Intel/neural-chat-7b-v3-3"
-default_num_cards=1
+default_model="meta-llama/Llama-2-70b-chat-hf"
```

Comment:
llama2 has license issue, so we are not allowed to use this. please change back to the original one.